src/STW.ProcessingApi.Function/Constants/ChedType.cs

```diff
@@ -2,6 +2,16 @@ namespace STW.ProcessingApi.Function.Constants;
 
 public static class ChedType
 {
-    public const string Chedpp = "CHED-PP";
-    public const string Chedp = "CHED-P";
+    public const string Cheda = "CHEDA";
+    public const string Chedp = "CHEDP";
+    public const string Chedd = "CHEDD";
+    public const string Chedpp = "CHEDPP";
+
+    public static readonly List<string> Values =
+    [
+        Cheda,
+        Chedp,
+        Chedd,
```

Comment:
Do we need CHEDA and CHEDD? Given you have added these to the values list and reference this in your common rules, these cert types can be used to trigger rules?

Reply:
I was under the impression these rules only apply to CHEDP and CHEDPP

Reply:
The orchestration layer needs to work for all certificate types